templates/core/terraform/variables.tf

```diff
@@ -35,12 +35,12 @@ variable "api_image_repository" {
 
 variable "api_app_service_plan_sku_tier" {
   type    = string
-  default = "PremiumV3"
+  default = "PremiumV2"
```

Comment:
question: sure it's fine, but why are we changing this?

Reply:
Because of quota issues in our subscriptions

Reply:
I've put it back